### DISCUSSION TITLE
desync kick does not work when host

### DISCUSSION THREAD

**Discussion by lonelybud** [YimMenu/YimMenu#3078] [Q&A] [answered]
i have tried to desync normal players when i am host and it does not work. Just wanted to confirm if it is a known issue or kind?

**Comment by D-Borkan** [comment]
the same, not work any more with me

**Comment by rkwapisz** [comment]
I don't recall desync ever working if you were the host.

**Reply by maybegreat48** [reply, marked as answer]
Yeah, it shouldn't work if you're the host
